### PR TITLE
[TASK] Correct reference to TypoScript manual

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

File: Documentation/ApiOverview/Assets/Index.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ tags.
 
 There are multiple ways to add assets to the :php:`PageRenderer` in TYPO3.
 For configuration options via TypoScript (usually used for the main theme files),
-see the :ref:`TypoScript Reference<t3tsref:setup-page-includecss-array`. In extensions,
+see the :ref:`TypoScript Reference <t3tsref:setup-page-includecss-array>`. In extensions,
```
Comment (mbrodala) [inline Documentation/ApiOverview/Assets/Index.rst:19]: I wonder if we can use the regular `ref` here ...

Comment (brotkrueml) [reply Documentation/ApiOverview/Assets/Index.rst:19]: Why? I rendered it and the link is displayed correctly now.

Comment (mbrodala) [reply Documentation/ApiOverview/Assets/Index.rst:19]: For consistency. We use `:ref:` everywhere so at least I wonder why we do something different here.

Comment (brotkrueml) [reply Documentation/ApiOverview/Assets/Index.rst:19]: I don't understand? I am using :ref:?

    :ref:`TypoScript Reference <t3tsref:setup-page-includecss-array>`

Comment (mbrodala) [reply Documentation/ApiOverview/Assets/Index.rst:19]: Yes, you are right. It's Monday morning.

**Review by mbrodala — APPROVED** [review]

**Pull request merged**